uwblueprint/marillac-place
# Authorization Fixes

## Notion ticket link

[Authorization Fixes](https://www.notion.so/uwblueprintexecs/Fix-authentication-in-the-frontend-90af88f3b6bd41baa98f0cc542a5d1d2)



## Implementation description
* Replaced jsonwebtoken library with jwtdecode
* Added a testapi component to validate the functionality of front end to backend api calling



## Steps to test
1. Pull changes
2. Run docker-compose up --build
3. Check to see if you can log in and go into the create entity page
4. Check the console to see if the required application is running



## What should reviewers focus on?
* 


## Checklist
